package.json

```diff
@@ -86,5 +86,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
-}
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
Why add this

Reply:
it gets added automatically if you have the latest version of yarn installed globally. So it detects which version is being used in the project and then autofills the property always with a sha included.

Anytime I run a `yarn ...` command it applies the change if the property does not exist but does not edit it if it does

Reply:
Maybe change your system yarn to v1 because I don't have this probelm

Reply:
turns out it's corepack not yarn but it's good to have for the repos that are using V4. I'll remove this

```bash
❯ yarn global -v
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager

1.22.22
```